### PR TITLE
Prepare First release

### PR DESCRIPTION
**Pull request by dmoranj**

### CODE REVIEW

File: rpm/SPECS/iotamanager.spec [hunk 1]
```diff
@@ -161,3 +161,27 @@ fi
 %{_install_dir}
 
 %changelog
+
+* Fri Sep 09 2016 Daniel Moran <daniel.moranjimenez@telefonica.com> 1.0F.0
```
Comment (fgalan) [inline rpm/SPECS/iotamanager.spec]: 1.0F.0?

File: lib/services/iotaRedirector.js [hunk 2]
```diff
@@ -248,7 +248,7 @@ function processRequests(req, res, next) {
 
                 callback(parseError, [response, parsedBody]);
             } else {
-                callback(parseError, [response, null]);
+                callback(null, [response, null]);
```
Comment (fgalan) [inline lib/services/iotaRedirector.js]: This is a bugfix? Worth to be mentioned in CNR?

Comment (dmoranj) [inline lib/services/iotaRedirector.js]: It was mentioned by the linter. It didn't fail in the tests. I don't think that's worth mentioning.

File: rpm/SPECS/iotamanager.spec (hunk 1 above)
Comment (dmoranj) [inline rpm/SPECS/iotamanager.spec]: Webstorm and OSX have a strange relationship; sometimes I'm trying to write in other window and it changes things without me realizing it. I'll fix it.

**Comment by fgalan** [comment]
LGTM

**Pull request merged**